e2e/kubectl.e2e.spec.ts

```diff
@@ -0,0 +1,117 @@
+import path from 'path';
+import {
+  ElectronApplication, BrowserContext, _electron, Page, Locator
+} from 'playwright';
+import { test, expect } from '@playwright/test';
+import { TestUtils } from './utils/TestUtils';
+import * as tools from './utils/ToolsUtils';
```

Comment:
```suggestion
import { kubectl } from './utils/ToolsUtils';
```
We only ever use `kubectl` in this file.